llvm/tools/llvm-exegesis/lib/Target.h

```diff
@@ -86,6 +86,16 @@ class ExegesisTarget {
                 ArrayRef<const char *> ValidationCounters,
                 const pid_t ProcessID = 0) const;
 
+  virtual bool checkOpcodeSupported(int Opcode,
+                                    const MCSubtargetInfo &SI) const {
+    return true;
+  }
+
+  // Find register by name, 0 if not found.
+  virtual unsigned findRegisterByName(const StringRef RegName) const {
+    return 0;
```

Comment:
Is it possible for a register to have the value 0? Why not `RISCV::NoRegister` for RISC-V? Thats what `RISCVAsmParser::matchRegisterNameHelper` returns when there is no match.

Reply:
Addressed